[maven export] write properties back to bndrun if successfully resolved

rotty3000 opened the pull request.

bjhargrave commented:
How does the title of this PR reflect the actual changes made by this PR? This PR has nothing to do with writing properties.

rotty3000 commented:
I updated the merge comment to reflect the text. the PR machine just takes the name of the last commit as the default title, I removed that commit from the PR, but github still stuck with the naming.
